examples/dashboards/kustomization.yaml

```diff
@@ -20,6 +20,9 @@ configMapGenerator:
 - name: grafana-controller-resources-metrics
   files:
   - ./controller-resources-metrics.json    
+- name: dns-operator
+  files:
+  - ./dns-operator.json    
```

Comment:
Would you mind updating 2 other places so the configmap will be mounted into grafana on both k8s and openshift?
https://github.com/Kuadrant/kuadrant-operator/blob/main/config/observability/grafana/grafana_deployment_patch.yaml
and
https://github.com/Kuadrant/kuadrant-operator/blob/main/config/observability/openshift/grafana/dashboards.yaml

The existing config in those files should hopefully be enough of an example what to add for the dns-operator configmap

Reply:
Ah, this is why it isn't working for me